src/components/Display/DisplaySettingsPopover.scss

```
@@ -4,13 +4,30 @@
   border: 1px solid #3a3a3a;
   border-radius: 8px;
   padding: 16px;
-  width: 280px;
+  // 314, not 280, because box-sizing below is now border-box: this rule read
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)